src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -32,6 +44,23 @@ class ClusterInfo:
     failures: str
 
 
+@dataclass
+class PipelineInfo:
+    pipeline_id: str
+    pipeline_name: str
+    creator_name: str
+    success: int
+    failures: str
+
+
+def azure_sp_conf_usage_check(config: str) -> bool:
```

Comment:
```suggestion
def _azure_sp_config_key_usage_check(key: str) -> bool:
```

please rename the method to be more clear with intent and also make it private